Content.Shared/SSDIndicator/SSDIndicatorSystem.cs

```diff
@@ -16,12 +23,46 @@ public override void Initialize()
     private void OnPlayerAttached(EntityUid uid, SSDIndicatorComponent component, PlayerAttachedEvent args)
     {
         component.IsSSD = false;
+
+        // Removes force sleep and resets the time to zero
+        if (_cfg.GetCVar(CCVars.ICSSDSleep))
+        {
+            component.FallAsleepTime = TimeSpan.Zero;
+            EntityManager.RemoveComponent<ForcedSleepingComponent>(uid);
```

Comment:
Won't this remove ForcedSleepingComponents from other sources too?

Reply:
well damn, firstly i wanted to do that but thought there will be a problem that cannot even happen
yeah, gonna fix that